java/vector/src/test/java/org/apache/arrow/vector/compare/TestRangeEqualsVisitor.java

```diff
@@ -26,13 +26,7 @@
 
 import org.apache.arrow.memory.BufferAllocator;
 import org.apache.arrow.memory.RootAllocator;
-import org.apache.arrow.vector.BigIntVector;
-import org.apache.arrow.vector.Float4Vector;
-import org.apache.arrow.vector.Float8Vector;
-import org.apache.arrow.vector.IntVector;
-import org.apache.arrow.vector.LargeVarCharVector;
-import org.apache.arrow.vector.VarCharVector;
-import org.apache.arrow.vector.ZeroVector;
+import org.apache.arrow.vector.*;
```

Comment:
nit: I realize this was probably the IDE but can this be reverted? For review it's easier to list out imports

Reply:
it is reverted now～